README.md

```diff
@@ -45,7 +45,15 @@ end
 By default, the `BasicErrorReportBuilder` will filter out password parameters from error reports sent to Bugsnag. You can customize this list inside your configuration:
 
 ```elixir
-config :plugsnag, :filter_parameters, ~w(password password_confirmation super_sekrit)
+config :plugsnag, :filter, ~w(password password_confirmation super_sekrit)
```

Comment:
See https://github.com/jarednorman/plugsnag/blob/master/lib/plugsnag/basic_error_report_builder.ex#L45 for current code